cmd/root_options.go

```diff
@@ -128,12 +129,12 @@ func RootOptionsLevelValidation(level validator.StructLevel) {
 		level.ReportError(opts.KernelConfigData, "kernelConfigData", "KernelConfigData", "required_kernelconfigdata_with_target_vanilla", "")
 	}
 
-	if opts.KernelVersion == 0 && (opts.Target == builder.TargetTypeUbuntuAWS.String() || opts.Target == builder.TargetTypeUbuntuGeneric.String()) {
+	if opts.KernelVersion == "" && opts.Target == builder.TargetTypeUbuntu.String() {
```

Comment:
In case we decide to keep ubuntu-generic and ubuntu-aws support, shouldn't they be listed here too?

Reply:
Oh good catch, let me add that back.